src/Views/Login/Login.js

```diff
@@ -1,11 +1,70 @@
-import React from 'react'
-
-const Login  = () => {
-    return ( 
-        <div>
-            Login
-        </div>
-      );
-}
- 
-export default Login;
\ No newline at end of file
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+const Login = ({ setAccessToken }) => {
+  const [emailInput, setEmailInput] = useState("");
+  const [passwordInput, setPasswordInput] = useState("");
+  const [error, setError] = useState();
+
+  const handleInputChange = (e) => {
+    if (e.target.id === "loginEmail") return setEmailInput(e.target.value);
+    if (e.target.id === "loginPassword")
+      return setPasswordInput(e.target.value);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    logIn(emailInput, passwordInput);
+    setEmailInput("");
+    setPasswordInput("");
+  };
+
+  const logIn = async (email, password) => {
+    let res = await fetch("https://pikabook-api.herokuapp.com/api/auth", {
+      method: "post",
+      headers: {
+        Accept: "application/json, text/plain, */*",
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ email, password }),
+    });
+    if (!res.ok) {
+      res = await res.text();
+      setError(res);
+    } else {
+      res = await res.json();
+      setAccessToken(res);
+    }
+  };
+
+  return (
+    <div>
+      <h1>Welcome to Pikabook!</h1>
+      <form onSubmit={handleSubmit}>
+        <input
+          type="text"
```

Comment:
I think for e-mail input type should be "email" instead of "text" so browser will suggest how to fill the field
```suggestion
          type="email"
```

> same for registration